io-engine/src/pool_backend.rs

```diff
@@ -234,6 +234,7 @@ pub trait IPoolProps {
     fn committed(&self) -> u64;
     fn md_props(&self) -> Option<PoolMetadataInfo>;
     fn encrypted(&self) -> bool;
+    fn max_expandable_size(&self) -> u64;
```

Comment:
And make lvm return None

```suggestion
    fn max_expandable_size(&self) -> Option<u64>;
```